### PR TITLE
VIZ - 74

### PR DESCRIPTION
**Pull request by vbojilova**
VIZ - 164 
Added ability to input multiple files at once and through a window selection

### CODE REVIEW

File: src/js/CreateAnalysis/Components/SelectedFileListPanel.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import SelectedFileList from "./SelectedFileList";
+
+import {inputConfig} from "../../../resources/config.js";
+const containerStyles = {
+  flexDirection: "row",
+  height: "50%",
+  backgroundColor: "#fdfdfd"
+};
+
+const SelectedFileListPanel = props => {
```
Comment (redpanda-cat) [inline src/js/CreateAnalysis/Components/SelectedFileListPanel.js]:
```suggestion
const SelectedFileListPanel = ({ filePaths, onDelete }) => {
```

File: src/resources/utils.js [hunk 2]
```diff
@@ -107,8 +111,12 @@ const lessThanMaxNumFile = param => {
 const isCorrectExt = args => {
   //Does the extension match the location it was dropped
   //Does the extension match the allowed ext
-  var allowedExt = inputConfig[args.target].extensions;
-  return allowedExt.indexOf(args.ext) > -1;
+  if (inputConfig.hasOwnProperty(args.target)) {
```
Comment (redpanda-cat) [inline src/resources/utils.js]:
```suggestion
return inputConfig.hasOwnProperty(args.target) && inputConfig[args.target]["extensions"].indexOf(args.ext) > -1
```

**Review by redpanda-cat — COMMENTED** [review]
Just some minor things

File: src/js/CreateAnalysis/Components/SelectedFileListPanel.js (hunk 1 above)
Comment (vbojilova) [reply src/js/CreateAnalysis/Components/SelectedFileListPanel.js]: I see this in SelectedFileList as well.. will change!

**Pull request merged**